veros/diagnostics/overturning.py

```diff
@@ -110,6 +141,13 @@ def diagnose(self, vs):
         sig_loc_face = 0.5 * (sig_loc[2:-2, 2:-2, :] + sig_loc[2:-2, 3:-1, :])
         trans = np.zeros((vs.ny + 4, self.nlevel), dtype=vs.default_float_type)
         z_sig = np.zeros((vs.ny + 4, self.nlevel), dtype=vs.default_float_type)
+
+        # Potential density using TEOS-10
+        self.prho[2:-2, 2:-2, :] += density.gsw.gsw_pot_rho_t_exact(vs,
+                                                vs.salt[2:-2, 2:-2, :, vs.tau],
```

Comment:
Wrong indentation. According to PEP8, this should align this line with the opening bracket from the line before. Alternatively, you can format it like this:

```python
self.prho[2:-2, 2:-2, :] += density.gsw.gsw_pot_rho_t_exact(
    vs,
    vs.salt[2:-2, 2:-2, : vs.tau],
    ...
)
```